packages/deno/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  extends: ['../../.eslintrc.js'],
+  ignorePatterns: ['lib.deno.d.ts'],
+  rules: {
+    '@sentry-internal/sdk/no-optional-chaining': 'off',
+    '@sentry-internal/sdk/no-nullish-coalescing': 'off',
+    '@sentry-internal/sdk/no-unsupported-es6-methods': 'off',
+    '@sentry-internal/sdk/no-class-field-initializers': 'off',
+  },
+  overrides: [
+    {
+      files: ['./test/*.ts'],
+      rules: {
+        'import/no-unresolved': 'off',
```

Comment:
When we get rid of all the `npm:` prefixes we can probably enable this rule again.

Reply:
Everything in `/test/*` is Deno typescript.
Some imports use denos URL imports which eslint doesn't understand

Reply:
Checks out!